src/main/scala/gitbucket/core/util/JGitUtil.scala

```diff
@@ -659,17 +660,44 @@ object JGitUtil {
     }
   }
 
+  def getCommitLog(git: Git, from: String, to: String): List[CommitInfo] = {
+    getCommitLog(git, Try(git.getRepository.resolve(from)).getOrElse(ObjectId.zeroId()), git.getRepository.resolve(to))
+  }
+
   /**
    * Returns the commit list between two revisions.
+   * This method takes strings as parameters, but does not define what happens when the strings are no valid
+   * commit ids. They can be either invalid syntactically, or they can be not within the repo.
    *
    * @param git the Git object
    * @param from the from revision
    * @param to the to revision
    * @return the commit list
    */
   // TODO swap parameters 'from' and 'to'!?
-  def getCommitLog(git: Git, from: String, to: String): List[CommitInfo] =
-    getCommitLogs(git, to)(_.getName == from)
+  def getCommitLog(git: Git, from: ObjectId, to: ObjectId): List[CommitInfo] =
```

Comment:
It seems that `git.getRepository.parseCommit()` needs to be called with `from` and `to` to support tags.